container/customrun.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
In the image we have installed `python`, can we use it instead of sh to run `artifactory-cleanup` command? It's easy to read python then sh

It's not important tho